app/routes/users/components/UserSettingsOAuth2.tsx

```diff
@@ -90,15 +91,20 @@ const UserSettingsOAuth2 = (props: Props) => {
     {
       dataIndex: 'delete',
       render: (id, grant) => (
-        <Tooltip
-          content="Fjern"
-          onClick={() => props.deleteOAuth2Grant(grant.id)}
-          style={{
-            marginTop: '-7px',
-          }}
+        <ConfirmModalWithParent
+          title="Bekreft handling"
+          message={`Er du sikker på at du vil fjerne token?`}
+          onConfirm={() => props.deleteOAuth2Grant(grant.id)}
+          closeOnCancel
         >
-          <Icon name="trash-outline" size={20} className={styles.deleteIcon} />
-        </Tooltip>
+          <Tooltip content="Fjern" style={{ marginTop: '-7px' }}>
+            <Icon
+              name="trash-outline"
+              size={18}
+              className={styles.deleteIcon}
+            />
+          </Tooltip>
+        </ConfirmModalWithParent>
```

Comment:
This seems like a nice change, but it's not really relevant to the PR? At least mention it in the description